utils/src/main/kotlin/storage/HttpFileStorage.kt

```diff
@@ -40,7 +40,7 @@ class HttpFileStorage(
     /**
      * The URL to store files at.
      */
-    private val url: String,
+    val url: String,
```

Comment:
I couldn't find a place where this is publicly used now, is it?

Reply:
It's used in `ScanResultsStorage:71` in a log message.